src/adapter/etl-adapter-json/src/Flow/ETL/Adapter/JSON/functions.php

```diff
@@ -34,6 +34,19 @@ function from_json(
     return $loader;
 }
 
+/**
+ * @param Path|string $path - string is internally turned into stream
```

Comment:
can we add a link to https://jsonlines.org/ that explains the difference? The content of this docblock is turned later automatically into the `from_json_lines` documentation just like this: 

https://flow-php.com/documentation/dsl/json/from-json/#dsl-function